compiler/rustc_borrowck/src/lib.rs

```diff
@@ -470,6 +471,8 @@ fn do_mir_borrowck<'tcx>(
 pub struct BodyWithBorrowckFacts<'tcx> {
     /// A mir body that contains region identifiers.
     pub body: Body<'tcx>,
+    /// The mir bodies of promoteds.
+    pub promoted: IndexVec<Promoted, Body<'tcx>>,
```

Comment:
Note that these may disappear soon: https://github.com/rust-lang/rust/pull/111693

Reply:
I've been following that PR as it would also simplify how we consume MIR. Should this PR wait until after #111693 is merged?

Reply:
afaict it will not be an issue for that PR to just remove this field here after a rebase